e2e/suite.go

```diff
@@ -17,15 +17,27 @@ import (
 	"testing"
 
 	"github.com/sylabs/singularity/e2e/actions"
+
```

Comment:
Why add all these whitespaces?

Reply:
Seeing if git will resolve the conflict automatically, theres always some conflict here. Same thing where the test are ran line 110++.